src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocCorrect.java

```diff
@@ -50,6 +50,11 @@ void foo9() {}
     void foo10() {}
 
     /**
+     * {@inheritDoc}mm
+     */
+    void foo9a() {}
+
+    /**
```

Comment:
please add cases like:
`{@inheritDoc} mm`, 
`{@inheritDoc} mm.`,
`{@inheritDoc} M m m m`,
`{@inheritDoc} M m m m.`
and
`mm{@inheritDoc}`, 
`mm. {@inheritDoc}`,
`M m m m {@inheritDoc}`,
` M m m m.{@inheritDoc}`